### ISSUE TITLE
CORS when using domain mapping

### ISSUE THREAD

**Issue by juslintek** [SuperPWA/Super-Progressive-Web-Apps#28]
To fix it need to create mu-plugin with code like this:
```
<?php
/**
 * Overriding custom SuperPWA functions before they are loaded for multisite
 */

// Exit if accessed directly
if ( ! defined( 'ABSPATH' ) ) {
	exit;
}

$pwa_plugin_name = 'super-progressive-web-apps/superpwa.php';
$pwa_plugin_dir  = ABSPATH . PLUGINDIR . '/' . $pwa_plugin_name;

include_once( ABSPATH . 'wp-admin/includes/plugin.php' );


if ( file_exists( $pwa_plugin_dir ) && is_plugin_active( $pwa_plugin_name ) ) {
	function superpwa_register_sw_mu() {

		$url = superpwa_sw('src');
		$path = parse_url($url, PHP_URL_PATH);
		$url_query = parse_url($url, PHP_URL_QUERY);
		$file_url = ($path ? $path : '') . ($url_query ? '?' . $url_query : '');

		wp_deregister_script( 'superpwa-register-sw' );
		wp_enqueue_script( 'superpwa-register-sw', SUPERPWA_PATH_SRC . 'public/js/register-sw.js', array(), null, true );
		wp_localize_script( 'superpwa-register-sw', 'superpwa_sw', array(
				'url' => $file_url,
			)
		);
	}

	add_action( 'wp_enqueue_scripts', 'superpwa_register_sw_mu', 20 );

}

```

**Comment by arunbasillal** [comment]
Thanks @juslintek 

#NoteToSelf - Related: https://wordpress.org/support/topic/superpwa-1-6-is-here-with-multisite-network-support/

**Comment by arunbasillal** [comment]
@aedelgod Thanks!

This is handled in 1.7 which will be released shortly.

**Comment by arunbasillal** [comment]
Hey guys, we have fixed this in version 1.7. Please test this out and confirm. Thanks.
